### ISSUE TITLE
Errors when vagrant up

### ISSUE THREAD

**Issue by shjiang** [creative-connections/Bodylight-VirtualMachine#8]
![image](https://user-images.githubusercontent.com/2359242/144841445-e07d12db-e07b-476f-b401-6049aa92b58a.png)

Can anyone help? Thanks a lot!

**Comment by TomasKulhanek** [comment]
The error comes from Virtualbox which reports `invalidarg` sent by VagrantUp tool. What version of Virtualbox do you use?

**Comment by shjiang** [comment]
> The error comes from Virtualbox which reports `invalidarg` sent by VagrantUp tool. What version of Virtualbox do you use?

6.1.30   r148432

**Comment by TomasKulhanek** [comment]
My combination `vagrant 2.2.14` and `virtualbox 6.1.30` is working on windows 10.  I see nonstandard `vagrant box add ...virtualbox.box --name westlife-eu/scientific_7_gui`. We're depended on Scientific Linux 7 minimal installation and westlife-eu/scientific_7_gui is one I managed in the past for several other projects. The error is very probably caused by your customized `box.ovf`, So I have limited possibilities to help you. I recommend to use by default the minimal image or the image with preinstalled binaries.

**Comment by shjiang** [comment]
> 

Thanks for your kind help. I will try again to follow your advice.  It is because the internet connection speed to download scientific_7_gui very very slow in the command mode, so I download the box from the vagrant website and install it manually.
Thanks again for your help.  This project is really helpful.

**Comment by TomasKulhanek** [comment]
In this case. Either do standard `cd Bodylight-VirtualMachine; vagrant up` and wait  or you may place the downloaded box somewhere and refer the file in `Vagrantfile` by changing the config.vm.box e.g.: `config.vm.box = "c:/Users/username/Downloads/bodylightvm.box"`

**Comment by shjiang** [comment]
I have tried serval times and failed in my computer. The errors were the same as follows, 
![image](https://user-images.githubusercontent.com/2359242/145049599-d918fb38-b76b-4794-86a6-ee57c780d3af.png)

However, I succeed in another computer. 
Still, I encountered another problem, and have no idea to handle it:
![image](https://user-images.githubusercontent.com/2359242/145050154-c0752037-fb17-4556-9277-362bd92f81b6.png)
and the log file is here
![image](https://user-images.githubusercontent.com/2359242/145050285-730d103b-5e2d-4a52-8055-ef8456c9af0d.png)

I use the latest vagrant and virtualbox, and everything seems ok before this step.

**Comment by shjiang** [comment]
> In this case. Either do standard `cd Bodylight-VirtualMachine; vagrant up` and wait or you may place the downloaded box somewhere and refer the file in `Vagrantfile` by changing the config.vm.box e.g.: `config.vm.box = "c:/Users/username/Downloads/bodylightvm.box"`

I followed your second advice, downloaded the box from this link,
"https://filedn.com/lHGc7w3H4jOpIe46u1nPt57/BodyligthVMImage/21.10/bodylightvm.box"
and modified `Vagrantfile`. It failed at the first time. When I looked at the message, it showed this, 
 
![image](https://user-images.githubusercontent.com/2359242/145064348-93612cf3-43f8-496c-abf8-8185460e7354.png)

Since I do not have a drive letter as “F”, I changed one of my drive letters to “F”. Now it works.
